.vscode/launch.json

```diff
@@ -14,44 +14,17 @@
 			],
 			"preLaunchTask": "npm: watch"
 		},
-		{
-			"name": "Web Extension",
-			"type": "extensionHost",
-			"debugWebWorkerHost": true,
-			"request": "launch",
-			"args": [
-				"--extensionDevelopmentPath=${workspaceFolder}",
-				"--extensionDevelopmentKind=web",
-				"${workspaceFolder}/sampleWorkspace"
```

Comment:
@artificial-ben (off topic) I'm curious to see how you set up your dev environment to create this extension. 

I didn't set any of these. I'm curious to see how they changed–hopefully sped up–your dev flow.